lya/data.py

```diff
@@ -29,34 +30,38 @@ def load_lya_data(filename=lya_data_path):
 
     data = Data(x, y, y_unc)
 
-    return data
-
-
-def plot_pk(params=None, filename=lya_data_path):
-    """Plot p(k) power spectrum data and theory, with panel for residuals
-    
-    Parameters
-    ----------
-    params : dict or None
-        Cosmological parameter dictionary; if None, then plot only data
-
 
-    Returns
-    -------
-    fig : matplotlib Figure instance
-    """
+    return data
 
-    data = load_lya_data(filename)
+def plot_pk(params=None, filename=lya_data_path, color='k',marker='o',ms=5,ls='',**kwargs):
+"""Plot p(k) power spectrum data and theory, with panel for residuals
 
-    if params is None:
-        fig, ax = plt.subplots(1, 1)
-        ax.errorbar(data.x, data.y, data.y_unc, color="k", marker="o", ms=5, ls="")
+Parameters
+----------
+params : dict or None
+    Cosmological parameter dictionary; if None, then plot only data
 
-        return fig
 
-    else:
-        ...  # TODO: assign this to Aryan
+Returns
+-------
+fig : matplotlib Figure instance
+"""
 
+if params is None:
```

Comment:
I think you're missing a `data = load_lya_data(filename)` line somewhere at the top of this function that was here before.